### PR TITLE
completed filter and sorting for list of products

### PR DESCRIPTION
**Pull request by ManikandanS910**

### CODE REVIEW

File: autozone-poc/src/Redux/reducers/productFilterReducer.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import {filterproducts} from '../../constants';
+
+export const productFilterReducer = (state={}, action) =>{
+    switch(action.type){
+        case filterproducts:
+            console.log("action type inside: " + action.type);
```
Comment (gokulwebdeveloper) [inline autozone-poc/src/Redux/reducers/productFilterReducer.js]: remove console.log

File: autozone-poc/src/Redux/reducers/productSortReducer.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import {sortproducts} from '../../constants';
+
+export const productSortReducer = (state={}, action) =>{
+    //console.log("action type : " + action.type);
+    switch(action.type){
+        case sortproducts:
+            console.log("action type inside: " + JSON.stringify(action.payload.sortType));
```
Comment (gokulwebdeveloper) [inline autozone-poc/src/Redux/reducers/productSortReducer.js]: remove all console.log from your files

File: autozone-poc/src/components/ProductDetails/DetailsSection.js [hunk 3]
```diff
@@ -222,9 +222,8 @@ const DetailsSection = (props) => {
                 responsiveRefreshRate={200}>
 
                 {currentProduct.product_image.map((image, index) => {
-                    let imagePath = image.replace("../","../src/utlis/");
                 return (
-                    <div key={index}><img src={imagePath} alt="" /></div>
+                    <div key={index}><img src={image} alt="" /></div>
```
Comment (gokulwebdeveloper) [inline autozone-poc/src/components/ProductDetails/DetailsSection.js]: try to add product name for alt tag

**Review by gokulwebdeveloper — CHANGES_REQUESTED** [review]

File: autozone-poc/src/Redux/reducers/productFilterReducer.js (hunk 1 above)
Comment (ManikandanS910) [reply autozone-poc/src/Redux/reducers/productFilterReducer.js]: Removed

File: autozone-poc/src/Redux/reducers/productSortReducer.js (hunk 2 above)
Comment (ManikandanS910) [reply autozone-poc/src/Redux/reducers/productSortReducer.js]: Removed logs

File: autozone-poc/src/components/ProductDetails/DetailsSection.js (hunk 3 above)
Comment (ManikandanS910) [reply autozone-poc/src/components/ProductDetails/DetailsSection.js]: Added product name for alt attribute

File: autozone-poc/src/utlis/clickAction/clickAction.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        handleClick(e) {
+            e.preventDefault();
+            console.log("common action clicked " + e.currentTarget.dataset.id);
+            console.log("common action  title " + e.currentTarget.dataset.title);
+
+
+
+            let category = "Brakes and Traction Control";
+
+            //this.props.filterProducts(props.productData);
```
Comment (gokulwebdeveloper) [inline autozone-poc/src/utlis/clickAction/clickAction.js]: can you remove commented lines

**Review by gokulwebdeveloper — CHANGES_REQUESTED** [review]

File: autozone-poc/src/utlis/clickAction/clickAction.js (hunk 4 above)
Comment (ManikandanS910) [reply autozone-poc/src/utlis/clickAction/clickAction.js]: Removed this unused file

**Pull request merged**